libmamba/src/core/query.cpp

```diff
@@ -621,7 +614,7 @@ namespace mamba
             else if (col.find_first_of(":") == col.npos)
             {
                 headers.emplace_back(col);
```

Comment:
Could the current code have problem as well? How about?
```suggestion
                headers.emplace_back(std::string(col));
```

Reply:
Converting `col` to `std::string` in `cmds.push_back(std::string(col));` is due to changing `cmds` to `std::vector<std::string>` (instead of `std::vector<std::string_view>`).
But it's not needed in `headers.emplace_back(col);` as `headers`(`std::vector<mamba::printers::FormattedString>`) already have a constructor accepting `std::string_view`.

Reply:
Sorry, my remark was not clear.

I was wondering whether we could also have this problem for all vectors referencing `std::string_view`s directly or indirectly, i.e. could we have the problem with `headers` again here if the original `std::string` of `std::string_view` are freed?

Reply:
Since `headers` is a vector of a struct (`FormattedString`) containing a `string` and not a `string_view`, it should be okay.
The only `string_view`s pointing to a local `string` being out of scope later (`sfmt`) were `cmds` and `args` which were modified to `string` now.
So seems ok so far.